packages/services/f-analytics/src/mixins/analytics.mixin.vue

```diff
@@ -93,7 +97,7 @@ export default {
         prepareAnalytics () {
             const platformData = { ...this.platformData };
 
-            platformData.name = this.$route.name;
+            platformData.name = MAP_ROUTE_TO_FEATURE_NAME[this.$route.name];
```

Comment:
Should we leave the option to add `this.$route.name` if the name is not in the `MAP_ROUTE_TO_FEATURE_NAME` object?

Reply:
Ah yeah that's a good shout. 👍

Reply:
Fixed.